source/MRMesh/MRPolylineRelax.cpp

```diff
@@ -0,0 +1,130 @@
+#include "MRBitSetParallelFor.h"
+#include "MRPolyline.h"
+#include "MRPolylineRelax.h"
```

Comment:
better put it on top of all to make sure that it is self-sufficient

Reply:
Fixed.